src/components/Accordion.tsx

```diff
@@ -1,5 +1,6 @@
-import React, { useState } from 'react';
-import styled from 'styled-components';
+import React, { useState } from "react";
+import styled from "styled-components";
+import { colors } from "../config/colors";
```

Comment:
@Agnieszka0544 its not very important but i rather use `styles` dir instead of `config`

Reply:
I found info that such files are placed in the 'config' dir, but you're right that name 'styles' is more appropriate. I corrected it